### DISCUSSION TITLE
Building Zephyr for Arty A7 100T

### DISCUSSION THREAD

**Discussion by marrrk** [zephyrproject-rtos/zephyr#36009] [Q&A]
Hi all, I've followed the tutorial on booting zephyr on the litex_vexriscv softcore up till the booting section. It seems as if everything loads fine but I get then get nothing after 'liftoff' - where I'd expert to see at least hello world. 

```
--============== Boot ==================--
Booting from serial...
Press Q or ESC to abort boot completely.
sL5DdSMmkekro
[FLTERM] Received firmware download request from the device.
[FLTERM] Uploading kernel (183932 bytes)...
[FLTERM] Upload complete (7.7KB/s).
[FLTERM] Booting the device.
[FLTERM] Done.
Executing booted program at 0x40000000

--============= Liftoff! ===============--

```

As the title says, I'm using the 100T variant as opposed to the 35T the tutorial mentions. I made sure to specify the part change when generating the gateware. I'm not quite sure where the problem could stem from. 

I'd appreciate any input!

**Comment by nashif** [comment]
@PiotrZierhoffer @mgielda do you have some hints here?

**Comment by yashi** [comment]
I don't know about litex, but with Arty A7 you can run Cortex M3 on it.  If you have Xilinx DesginStart, the following should work:  

```
$ west build -b arty_a7_arm_designstart_m3 -t flash zephyr/samples/hello_world
```
At least you can check your hardware works or not.
